src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1

```diff
@@ -0,0 +1,120 @@
+function Copy-CohesityView {
+    <#
+        .SYNOPSIS
+        Clones the specified Cohesity View.
+        .DESCRIPTION
+        Clones the specified Cohesity View.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Copy-CohesityView -TaskName "Task-clone-a-view" -SourceViewName "source-view" -TargetViewName "target-view" -TargetViewDescription "Create a view clone" -QosPolicy "Backup Target Low" -JobId 12345
```

Comment:
If backward compatibility is not at stake get rid  of jobId, let the function figure out the backup job and clone view from that one .

Reply:
The user need to specify the job id to create a clone. The view could be cloned by multiple jobs. The parameters are kept unchanged from the C# based cmdlet.